.github/workflows/build.yml

```diff
@@ -44,13 +44,16 @@ jobs:
       - name: Copy plugin binary to standard location
         shell: bash
         run: cp target/${{ matrix.config.target }}/release/pluginify${{ matrix.config.extension }} target/release/pluginify${{ matrix.config.extension }}
+      - name: Duplicate plugin binary for self-pluginification
+        shell: bash
+        run: cp target/release/pluginify${{ matrix.config.extension }} target/release/self-pluginify${{ matrix.config.extension }} 
 
       # We cannot self-pluginify on aarch64 because binaries won't run on the runner
       # (the runner is always x86).  For other plugins, that wouldn't be an issue.
       # So we'd like to set the merge pattern to fit that scenario.  So... we'll cheat.
       - name: Pluginify - self-pluginify
         if: ${{ matrix.config.arch == 'amd64' }}
-        run: ./target/release/pluginify
+        run: ./target/release/self-pluginify
```

Comment:
Quick fix for the build here, but is it bad if I do it for any amd64 instead of just windows?